### ISSUE TITLE
"Data Preview" for DiskANN index doesn't work

### ISSUE THREAD

**Issue by yhmo** [zilliztech/attu#333]
**Describe the bug:**
"Data Preview" for DiskANN index doesn't work.

**Steps to reproduce:**
1. create a collection with DiskANN index
```
import random
import time
import numpy as np

from pymilvus import (
    connections,
    FieldSchema, CollectionSchema, DataType,
    Collection,
    utility,
)

connections.connect(host='localhost', port='19530')
print(utility.get_server_version())

collection_name = "test"
dim = 128
metric_type = "L2"

# create collection
id_field = FieldSchema(name="id", dtype=DataType.INT64, is_primary=True, auto_id=False,)
vector_field = FieldSchema(name="vector", dtype = DataType.FLOAT_VECTOR, dim=dim)
name_field = FieldSchema(name="name", dtype = DataType.VARCHAR, max_length=256)

schema = CollectionSchema(fields=[id_field, vector_field, name_field])

if utility.has_collection(collection_name):
    utility.drop_collection(collection_name)

collection = Collection(name=collection_name, schema=schema)
print(f"Collection '{collection_name}' created")

batch_size = 10000
data = [
    [k for k in range(batch_size)],
    [[random.random() for _ in range(dim)] for _ in range(batch_size)],
    [f"name_{k}" for k in range(batch_size)]
]
for i in range(3):
    print("insert batch", i)
    collection.insert(data)

collection.flush()
print(collection.num_entities)

index_params = {
    'metric_type': metric_type,
    'index_type': "DISKANN",
    'params': {},
}
collection.create_index(field_name="vector", index_params=index_params)
print("index created")

collection.load()
print("collection loaded")
```

2. Open the Attu dashboard
3. Choose the collection and click "Data Preview"

The error shows "Param 'search_list_size'(20) is not in range [100, 4294967295] err %!w(<nil>)"

**Attu version:**
Milvus v2.2.14
Attu v2.2.8

**Attu version:**

**Comment by yhmo** [comment]
![Unsaved Image 1](https://github.com/zilliztech/attu/assets/2282099/b269b1d8-135e-402a-b779-052f88f41f87)

**Comment by yhmo** [comment]
The root cause is:
https://milvus.io/docs/disk_index.md#Prerequisites  in this doc, it says "search_list" range is [top_k, min( 10 * top_k, 65535)] for k > 20 and [top_k, 200] for top_k <= 20
But the Attu passes search_list:20 and topk:100 to the server.

Change the search_list to 100 can fix this error.

**Comment by shanghaikid** [comment]
yeah. I will fix this in the next release.

**Comment by shanghaikid** [comment]
> Great, thanks! Would it be possible, to fix it in the Version v2.2.8, since it is more suitable for Milvus v2.2.14? https://discordapp.com/channels/1160323594396635310/1183736071234793494/1183986773081198672



Why not just use the latest version, if you have any issue, just let me know, I will fix it.
